hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncRpcRetryingCallerFactory.java

```diff
@@ -119,8 +119,8 @@ public AsyncRpcRetryingCallerFactory(AsyncConnectionImpl conn, Timer retryTimer)
       return this;
     }
 
-    public SingleRequestCallerBuilder<T> pauseForCQTBE(long pause, TimeUnit unit) {
-      this.pauseForCQTBENs = unit.toNanos(pause);
+    public SingleRequestCallerBuilder<T> pauseForServerOverloaded(long pause, TimeUnit unit) {
```

Comment:
this class is IA.Private, so I simply rename all of these methods rather than handle deprecation